hasher-matcher-actioner/hmalib/indexers/tests/test_lcc_indexer.py

```diff
@@ -26,8 +28,13 @@ def test(self):
             )
             tbi.force_flush()
             test_class = LCCIndexer()
-            test_index = test_class.build_index_from_last_24h(
-                tbi.type, tbi.storage_path, tbi.bucket_width
-            )
+            with metrics.timer(metrics.names.lcc.build_index):
```

Comment:
blocking: The metrics timer should be used inside the LCCIndexer class.

In what we have here, only the test ends up writing metrics.